### DISCUSSION TITLE
AI Deck WiFi Image Streamer Demo Fails Upon Starting up Push Demo on Crazyflie

### DISCUSSION THREAD

**Discussion by Isenf1er** [bitcraze/discussions#1300] [Q&A]
Hi,

I am currently working with the AI Deck, Flow Deck V2, and Multi-Ranger, but I am encountering issues with getting all three boards to communicate effectively together. 

Right now, my goal is to be able to run two given demos, the push demo and the wifi image streamer demo given by Bitcraze at the same time, but I am encountering an issue where the WiFi access point will not be discoverable upon activation of the push demo.

To be more explicit, the push demo is flashed already to the Crazyflie STM32 with the result that it works completely as intended. The wifi image stream example is flashed to the GAP8 on the AI deck and works completely fine as well. The issue occurs when starting the wifi example in VScode, the video stream tab opens and displays properly, but as soon as I activate the push demo on the crazyflie, the video stream closes itself and I am unable to access the wifi access point so long as the push demo is active.

All firmware is updated to the latest possible package including the crazyflie firmware and the AI deck firmware

**Comment by gemenerik** [comment]
This may be because the configuration used by the app and the non-app firmware is not identical. That is, if you configure with `make menuconfig` in the root dir of the firmware, this does configuration does not apply to the app. See #1284. That would explain why when flashing the app you lose your access point configuration.
